toponetx/utils/normalization.py

```diff
@@ -1,5 +1,6 @@
-"""Normalize of adjacency, boundary matrices of complexes."""
+"""Normalize of Laplacians, (co)adjacency, boundary matrices of complexes."""
 
+from typing import Tuple, Union
```

Comment:
Please don't use `Tuple` and `Union` from the typing module:
- Instead of `typing.Tuple`, use the built-in `tuple`. It works exactly the same.
- Instead of `Union[A, B]` use the union type expression `A | B`.